packages/amplify-e2e-tests/src/__tests__/pinpoint/android-analytics-pinpoint-config.test.ts

```diff
@@ -0,0 +1,35 @@
+import { createNewProjectDir, deleteProject, deleteProjectDir, initAndroidProjectWithProfile } from '@aws-amplify/amplify-e2e-core';
+import { androidValidate, runPinpointConfigTest } from './analytics-pinpoint-config-util';
+
+const envName = 'integtest';
+const projectSettings = {
+  envName,
+  disableAmplifyAppCreation: false,
+};
+const frontendConfig = {
+  frontend: 'android',
+  config: {
+    ResDir: 'app/src/main/res',
+  },
+};
+
+describe('android analytics pinpoint configuration', () => {
+  let projectRoot: string;
+
+  beforeEach(async () => {
+    projectRoot = await createNewProjectDir('notifications-pinpoint');
+  });
+
+  afterEach(async () => {
+    deleteProjectDir(projectRoot);
+  });
+
+  it('should add pinpoint to Android configuration', async () => {
+    await initAndroidProjectWithProfile(projectRoot, projectSettings);
+    try {
+      await runPinpointConfigTest(projectRoot, envName, frontendConfig, androidValidate);
+    } finally {
+      await deleteProject(projectRoot);
```

Comment:
should this go in the afterEach function? (same question for the other tests below)

Reply:
oh, good catch, initially I had all the tests in the same file and wanted to keep the initialization and deletion at the same level, now since I split them I will move to the before/after each